packages/wonder-blocks-dropdown/components/multi-select-menu.md

```diff
@@ -0,0 +1,241 @@
+The multi select menu allows the selection of multiple items. Clients are
+responsible for keeping track of the selected items in the menu.
+
+TODO(sophie): keyboard navigation and dismissal
+
+The first menu starts with nothing selected and has no selection shortcuts.
+There is also a disabled item, and this menu has a set width.
+
+The second menu starts with one item selected and has selection shortcuts for
+select all and select none. The second menu does not have a predefined placeholder.
```

Comment:
I noticed that the items in these menus have white borders on the left/right whereas the ones in the SingleSelectMenu do not.  These shouldn't have any white borders on the left/right either.

Reply:
The regular menu items look good now, but the shortcuts still have a slight white border on the left/right.